From 5dd2bd725061ccf1db4fac7f7b1f7c2131af54d3 Mon Sep 17 00:00:00 2001
Message-Id: <5dd2bd725061ccf1db4fac7f7b1f7c2131af54d3.1368111914.git.minovotn@redhat.com>
In-Reply-To: <405603258af5154387bea676be1f904b6713f6ae.1368111913.git.minovotn@redhat.com>
References: <405603258af5154387bea676be1f904b6713f6ae.1368111913.git.minovotn@redhat.com>
From: Amit Shah <amit.shah@redhat.com>
Date: Wed, 24 Apr 2013 08:17:59 +0200
Subject: [PATCH 25/65] char: rename qemu_chr_get_msgfd() ->
 qemu_chr_fe_get_msgfd()

RH-Author: Amit Shah <amit.shah@redhat.com>
Message-id: <d1823120913adc1f680ca22011ac58299f404a52.1366724981.git.amit.shah@redhat.com>
Patchwork-id: 50803
O-Subject: [RHEL6.5 qemu-kvm PATCH 25/65] char: rename qemu_chr_get_msgfd() -> qemu_chr_fe_get_msgfd()
Bugzilla: 909059
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

From: Anthony Liguori <aliguori@us.ibm.com>

Signed-off-by: Anthony Liguori <aliguori@us.ibm.com>
(cherry picked from commit 74c0d6f02048767f05ba671f1b5ca85aad720446)

Signed-off-by: Amit Shah <amit.shah@redhat.com>

Conflicts:
	hw/ivshmem.c

Signed-off-by: Amit Shah <amit.shah@redhat.com>
---
 monitor.c   | 2 +-
 qemu-char.c | 2 +-
 qemu-char.h | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 monitor.c   | 2 +-
 qemu-char.c | 2 +-
 qemu-char.h | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/monitor.c b/monitor.c
index 7cd5aeb..3d81bf7 100644
--- a/monitor.c
+++ b/monitor.c
@@ -2759,7 +2759,7 @@ static int do_getfd(Monitor *mon, const QDict *qdict, QObject **ret_data)
     mon_fd_t *monfd;
     int fd;
 
-    fd = qemu_chr_get_msgfd(mon->chr);
+    fd = qemu_chr_fe_get_msgfd(mon->chr);
     if (fd == -1) {
         qerror_report(QERR_FD_NOT_SUPPLIED);
         return -1;
diff --git a/qemu-char.c b/qemu-char.c
index 4764a2d..d7a9c0a 100644
--- a/qemu-char.c
+++ b/qemu-char.c
@@ -164,7 +164,7 @@ void qemu_chr_be_write(CharDriverState *s, uint8_t *buf, int len)
     s->chr_read(s->handler_opaque, buf, len);
 }
 
-int qemu_chr_get_msgfd(CharDriverState *s)
+int qemu_chr_fe_get_msgfd(CharDriverState *s)
 {
     return s->get_msgfd ? s->get_msgfd(s) : -1;
 }
diff --git a/qemu-char.h b/qemu-char.h
index 401de83..04eea6f 100644
--- a/qemu-char.h
+++ b/qemu-char.h
@@ -97,7 +97,7 @@ int qemu_chr_fe_ioctl(CharDriverState *s, int cmd, void *arg);
 void qemu_chr_generic_open(CharDriverState *s);
 int qemu_chr_be_can_write(CharDriverState *s);
 void qemu_chr_be_write(CharDriverState *s, uint8_t *buf, int len);
-int qemu_chr_get_msgfd(CharDriverState *s);
+int qemu_chr_fe_get_msgfd(CharDriverState *s);
 void qemu_chr_accept_input(CharDriverState *s);
 void qemu_chr_info_print(Monitor *mon, const QObject *ret_data);
 void qemu_chr_info(Monitor *mon, QObject **ret_data);
-- 
1.7.11.7

